daemon/src/projection.rs

```diff
@@ -527,16 +538,30 @@ impl From<Quote> for bitmex_price_feed::Quote {
     }
 }
 
+// TODO: Rename this to Order (or even better: both model::Order and this to Offer)
 #[derive(Debug, Clone, PartialEq, Serialize)]
 pub struct CfdOrder {
     pub id: OrderId,
 
     pub trading_pair: TradingPair,
     pub position: Position,
 
+    /// The maker's price for opening a position
     #[serde(with = "round_to_two_dp")]
     pub price: Price,
 
+    /// Fee charged by the maker for opening a position
+    ///
+    /// The opening feel
+    pub opening_fee: Amount,
+
+    /// Funding fee that is collected for the CFD period (until the CFD expires)
+    ///
+    /// The fee that is charged upon every funding round, which is currently charged on every
+    /// roll-over. The fee represents the interest payed from the long position to the short
+    /// position. At the moment the period is 1hour (because rollover is one hour).
+    pub funding_fee_per_period: Amount,
```

Comment:
I think we might additionally need to define the `funding_round_interval` (or similar name), that expresses how often  `funding_fee_per_period` is collected. 

This should be equal to `settlement_interval`, but it is currently not, until we resolve discussion:

Reply:
added.